Utils/azuretools-core/src/com/microsoft/azuretools/authmanage/Environment.java

```diff
@@ -56,6 +58,8 @@ public AzureEnvironment getAzureEnvironment() {
         }
     };
 
+    public static final List<Environment> ENVIRONMENT_LIST = ImmutableList.of(GLOBAL, CHINA, GERMAN, US_GOVERNMENT);
```

Comment:
Can  we still configure the environment from Adaprovide.json?

Reply:
No,it will consult environment from cli directly.(But there are some issues in this implement)

Reply:
Please mark the issues in the code comment for sdk team.

Reply:
Tracked in https://github.com/Azure/autorest-clientruntime-for-java/issues/673